daemon/cmd/vitals.go

```diff
@@ -52,7 +43,7 @@ func (d *Daemon) getHealthReport() models.ModulesHealth {
 
 func toModuleHealth(m cell.Status) *models.ModuleHealth {
 	return &models.ModuleHealth{
-		ModuleID:    m.ModuleID,
+		ModuleID:    string(m.ModuleID),
```

Comment:
nit: This is going to be common usage and since we now have a type why not make it a Stringer?

Reply:
Hmm now that you mentioned this, I think I'll go back to what I had before with `type FullModuleID []string` and with a `String()`, and then use `String()` as map keys here.